components/outputs/plot-altair/index.qmd

```diff
@@ -0,0 +1,220 @@
+---
+title: Plot (Altair)
+sidebar: components
```

Comment:
Altair doesn't seem to be showing up in the sidebar

<img width="1090" height="874" alt="Screenshot 2025-12-04 at 2 14 19 PM" src="https://github.com/user-attachments/assets/eb4d67e9-a0fa-449b-84e6-44ada49dad19" />